### ISSUE TITLE
Uncaught TypeError: domNode.replaceData is not a function

### ISSUE THREAD

**Issue by nikoudel** [elm/virtual-dom#148]
Hi! I'm having an issue which might belong here.

![image](https://user-images.githubusercontent.com/1636683/48833848-f6fc5c80-ed84-11e8-800f-29758b2e2ddb.png)

First, the app fails to render correctly - it's supposed to be like this:

![image](https://user-images.githubusercontent.com/1636683/48834155-b6e9a980-ed85-11e8-9610-896de45104fc.png)

Second, the app goes into an infinite loop when I click any of the links (the console errors).

This happens with a fresh Elm 0.19 install and the code from the example app on this guide [page](https://guide.elm-lang.org/webapps/navigation.html).

elm.json looks like this:
```
{
    "type": "application",
    "source-directories": [
        "src"
    ],
    "elm-version": "0.19.0",
    "dependencies": {
        "direct": {
            "elm/browser": "1.0.1",
            "elm/core": "1.0.2",
            "elm/html": "1.0.0",
            "elm/url": "1.0.0"
        },
        "indirect": {
            "elm/json": "1.1.2",
            "elm/time": "1.0.0",
            "elm/virtual-dom": "1.0.2"
        }
    },
    "test-dependencies": {
        "direct": {},
        "indirect": {}
    }
}
```

**Note:** the problem only appears in a certain Google Chrome instance (but not in IE or Firfox). Also the app works correctly when I run it in incognito mode in the same Chrome instance. And also it runs fine in another computer with the same Google Chrome version (70.0.3538.110) and the same addons. Disabling the addons doesn't help. Any ideas?

**Comment by bitfella** [comment]
Hi there. Do you have JSON Viewer Chrome extension installed? In my case that's what causes the issue.
Disabled it and everything works just fine.

**Comment by nikoudel** [comment]
You are right! Disabling that extension helped. Didn't notice I had that extension installed on my main PC but not on the other.

I don't know though should I close this ticket or not... it would be kinda lame to tell app users to disable their extensions so they could use my site...

**Comment by bitfella** [comment]
I suppose you are using Elm Reactor to test that code, as suggested in the docs. You should generate production code then and minify it. I guess production-ready code should not be affected by the kind of issue you opened. Give it a try!

**Comment by nikoudel** [comment]
Nope, that didn't work. I used uglifyjs as described [here](https://elm-lang.org/0.19.0/optimize) and updated the html file to load the minified version but enabling the JSON Viewer extension still breaks it.

**Comment by nikoudel** [comment]
This issue is most likely caused by this [bug](https://github.com/tulios/json-viewer/issues/203) in JSON Viewer.

A workaround is avoiding raw text without enclosing html elements, i.e. using `span [] [ text "The current URL is: " ]` instead of `text "The current URL is: "`.

Leaving for others to decide whether the Uncaught TypeError and the infinite loop thing should be still prevented in Elm.

**Comment by Natim** [comment]
Apparently Grammarly does that

**Comment by kubukoz** [comment]
Dark Reader too. There goes a few hours of work...
![image](https://user-images.githubusercontent.com/894884/69168595-42ac2800-0af7-11ea-90ee-31ec49f5534f.png)
